### ISSUE TITLE
So what has your experience been with devcontainers and dotfiles?

### ISSUE THREAD

**Issue by shadycuz** [felipecrs/dotfiles#79]
I'm looking to start using a dotfile repo, mainly because I use so many devcontainers now and I'm sick of not having a consistent shell. How is chezmoi?

**Comment by felipecrs** [comment]
Hi @shadycuz 👋!

My experience with devcontainers and dotfiles has been great! Chezmoi is awesome!

Little demo:

https://user-images.githubusercontent.com/29582865/174459272-ef7ed06f-ef33-46bf-8271-6d97072c9e63.mp4

**Comment by shadycuz** [comment]
Oh yeah thats super sweet =).

I'm wondering if I should have Chezmoi call Ansible to do my machine bootstrapping, not my dev containers but my actual machine. Instead of trying to make my install scripts idempotent.

**Comment by felipecrs** [comment]
Bro, I spent countless hours because I didn't do that. I think I would do that, if I were to start now.

**Comment by shadycuz** [comment]
What did you do for visual studio code? Are you still setting the file extension stuff to helm?

**Comment by felipecrs** [comment]
Yes, for `yaml` files I'm using the Helm Intellisense. For everything else, I try to put the template markers in comment blocks so it does not affect the whole file syntax.

Like this:

https://github.com/felipecrs/dotfiles/blob/c1493f385fee7981c435580771f314df4f3ceaae/root/.chezmoiscripts/run_after_10-install-apt-packages.sh.tmpl#L31-L40
